src/redisearch_api.h

```diff
@@ -90,6 +91,61 @@ struct RSIdxOptions {
   RSLanguage lang;
 };
 
+struct RSIdxField {
+  char *path;
```

Comment:
Wonder if we should version the struct for future changes and backward comparability

Reply:
Isn't that a concern when loading/saving to rdb?

Reply:
Added `RS_INFO_CURRENT` and `RS_INFO_INIT` as version flags for support in future versions.